### DISCUSSION TITLE
Document of sequence seems wrong

### DISCUSSION THREAD

**Discussion by Lannister-Xiaolin** [BehaviorTree/BehaviorTree.CPP#831] [Q&A]
![image](https://github.com/BehaviorTree/BehaviorTree.CPP/assets/37075966/85cca379-a0ab-4320-80ab-1191652f44dd)

**Comment by nbbrooks** [comment]
Link to page https://www.behaviortree.dev/docs/nodes-library/sequencenode/

I agree, I would change it to "FAILURE"

**Comment by nbbrooks** [comment]
Perhaps that column is meant to describe what happens if there is a `RetryUntilSuccessful` decorator as a parent to the Sequence* block (shown in the SequenceWithMemory example)?
